tests/conftest.py

```diff
@@ -212,6 +213,13 @@ def pytest_collection_modifyitems(config, items: List[pytest.Item]):
             if item.get_closest_marker("pydanticgen_npd"):
                 item.add_marker(skip_npd)
 
+    # skip docker tests when docker server not present on the system
+    if not _docker_server_running():
+        skip_docker = pytest.mark.skip(reason="Docker server not running on host machine")
+        for item in items:
+            if item.get_closest_marker("docker"):
+                item.add_marker(skip_docker)
+
```

Comment:
With this configuration, we would not be testing against Kroki on CI Windows machines... what is probably a very good idea. But just to be sure that we are aware of the consequences.

Reply:
into it!

Reply:
imo the goal is to be like "by all means plantuml ppl implement as rigorous of tests as you want but if the tests are flaky please can we avoid that impeding other work?"

Reply:
(it's why i did this, because i didn't want to be the next source of "oh dang it's breaking again", i think that's a good precedent to set https://github.com/linkml/linkml/pull/2178#issuecomment-2375514360 )